RecaptchaValidation/Controllers/RecaptchaController.cs

```diff
@@ -19,22 +21,20 @@ public RecaptchaController(IRecaptchaService recaptcha, IOptions<RecaptchaOption
             _recaptcha = recaptcha;
         }
 
-        [HttpPost("Verify")]
-        public async Task<IActionResult> Verify([FromQuery] string RecaptchaToken)
+        [HttpPost("verify")]
+        public async Task<IActionResult> Verify([FromQuery] string recaptchaToken)
         {
-            IRecaptchaRequestMessage request = new RecaptchaRequestMessage(RecaptchaToken, HttpContext.Connection.RemoteIpAddress.ToString(), _config);
+            var request = new RecaptchaRequestMessage(recaptchaToken, HttpContext.Connection.RemoteIpAddress.ToString(), _config.Value.Keys.Secret, _config.Value.VerifyUrl);
 
-            _recaptcha.InitializeRequest(request);
-
-            IRecaptchaResponseMessage response = await _recaptcha.Execute();
+            RecaptchaResponseMessage response = await _recaptcha.Execute(request);
 
             if(!response.success)
```

Comment:
properties on a class should be PascalCased

Reply:
I'm only not changing it because from what I've noticed, is if it's a dependency injected via the constructor it's been our standard to prefix with underscore